examples/air-quality-sensor-app/linux/README.md

```diff
@@ -136,10 +136,11 @@ $ echo '{"Name":"TemperatureMeasurement","NewValue":1800}' > /tmp/chip_air_quali
 
 ### Trigger Humidity change event
 
-Generate event `RelativeHumidityMeasurement`, to change the temperate value.
+Generate event `RelativeHumidityMeasurement`, to change the temperate value
```

Comment:
Pre-existing, but what is "the temperate value" and how is it changed by generating a RelativeHumidityMeasurement event?

Reply:
Fixed by [1f5d07c](https://github.com/project-chip/connectedhomeip/pull/32254/commits/1f5d07c6662bb203d7bfec2123af830f4ef55268)